README.md

```diff
@@ -50,7 +50,7 @@ https://user-images.githubusercontent.com/25933231/130909536-e096d342-19d5-4139-
    		displayTime: true,
    		displayTimeline: true,
    		displayClockSymbol: true,
-   		displayOnlyOnRain: false,
+   		displayHoursBeforeRain: Infinity,
```

Comment:
Even if Infinity is syntactically correct, could we use -1 for "show always" instead?

Reply:
Yes, good idea